prac_05/emails.py

```diff
@@ -0,0 +1,38 @@
+"""
+CP1404/CP5632 Practical 05
+Emails
+Estimate: 25 minutes
+Actual:  minutes
```

Comment:
Must have forgotten to update how long it actually took

Reply:
Yep noted. I remember writing it down on paper. Forgot to log it on the comment :facepalm: